src/core/packages/saved-objects/utils-server/src/saved_objects_utils.ts

```diff
@@ -109,4 +113,37 @@ export class SavedObjectsUtils {
   ): SavedObjectMigrationFn<InputAttributes, MigratedAttributes> {
     return isFunction(migration) ? migration : migration.transform;
   }
+
+  public static getName(
+    savedObject?: Pick<SavedObject<unknown>, 'attributes'> | null,
+    nameAttribute?: string
+  ): string | undefined {
+    if (!savedObject) {
+      return undefined;
+    }
+
+    const attributes = savedObject?.attributes as Record<string, unknown> & {
+      name?: string;
+      title?: string;
+    };
+
+    const fallbackTitle = attributes?.name ?? attributes?.title ?? '';
```

Comment:
prefer explicit (if somewhat verbose) code that's easier to read for all developers.
for example:
```suggestion
   const fallbackTitle = attributes?.name || attributes?.title || '';
```

Reply:
Pushed in c3071cc846dd900af3da75cb478e897a13e3da8f